pr_agent/git_providers/github_provider.py

```diff
@@ -26,6 +26,8 @@ def set_pr(self, pr_url: str):
         self.pr = self._get_pr()
 
     def get_files(self):
+        if hasattr(self.pr, 'files'):
```

Comment:
The 'get_files' method now has a condition to check if 'files' attribute exists in 'pr' object. It would be better to ensure that 'files' attribute always exists in 'pr' object (it can be an empty list if there are no files). This way, you can avoid the conditional check here and make the code cleaner. [medium]